engines/decidim_hospitalet-surveys/app/commands/decidim_hospitalet/surveys/admin/create_survey_result.rb

```diff
@@ -89,6 +89,13 @@ def existing_user
           ).first
         end
 
+        def default_user
+          Decidim::User.where(
```

Comment:
Yeah, I know, that's why I say we should better wait until this happens more often (if it ever does so).